core/unit_test/CMakeLists.txt

```diff
@@ -341,6 +341,9 @@ if(Kokkos_ENABLE_PROFILING)
     tools/printing-tool.cpp
   )
 
+  # FIXME_HIP
```

Comment:
Wait no lets not do this. Can we please just use the Kokkos_ARCH_VEGA906 or whatever option and add that to the nightly test?

Reply:
I just went with `KOKKOS_ADD_LIBRARY` instead.